tests/unit/store.spec.ts

```diff
@@ -3,9 +3,18 @@ import getters from '@/store/getters';
 import mutations from '@/store/mutations';
 import { emptyState, VQBState } from '@/store/state';
 
+function stateWithOnePipeline(pipeline: Pipeline) {
```

Comment:
What about
```suggestion
function stateWithOnePipeline(pipeline: Pipeline, customProperties = {}) {
```

and then merge initial pipeline with `customProperties`?

Reply:
I personally feel that brings nothing helpful compared to a simple splat. My policy would be: if a simple syntax feature, clear to read, is there, I'd prefer the language feature to a custom function. (It also seems to be our current policy towards other utilities like lodash).  
Could you develop on which errors it could prevent, or on how simpler to use if could be? Very open to challenge my policy regarding that :)

Reply:
No strong opinion about that, it's just that it's a classical pattern to have a parameter to override defaults. Besides, it would mimic the `setupStore()`